assembl/models/attachment.py

```diff
@@ -482,4 +482,8 @@ class VoteSessionAttachment(Attachment):
         'with_polymorphic': '*'
     }
 
-    # TODO: permissions !!!
+    crud_permissions = CrudPermissions(
+        P_ADMIN_DISC,   # create
```

Comment:
I think that you can use named attributes instead of comments if you want it to be explicit (and I think it is a good idea to be explicit (even more with the 7 arguments that we have elsewhere in the code)

Reply:
agree, thanks